### ISSUE TITLE
Release version for slick 3.5.0

### ISSUE THREAD

**Issue by amumurst** [tminglei/slick-pg#703]
I guess its coming through scala-steward at any point. Would be good to get a non-M release of slick-pg also for that :-) https://github.com/slick/slick/discussions/2908

**Comment by andyczerwonka** [comment]
3.5.1 has been released as well, so it'd be nice to cut a release please.

```
[error]         * com.typesafe.slick:slick_2.13:3.5.1 (pvp) is selected over {3.5.0, 3.4.1}
[error]             +- com.byteslounge:slick-repo_2.13:1.8.1              (depends on 3.5.1)
[error]             +- com.typesafe.slick:slick-hikaricp_2.13:3.5.1       (depends on 3.5.1)
[error]             +- com.github.tminglei:slick-pg_json4s_2.13:0.21.1    (depends on 3.4.1)
[error]             +- com.github.tminglei:slick-pg_core_2.13:0.21.1      (depends on 3.4.1)
[error]             +- com.github.tminglei:slick-pg_circe-json_2.13:0.21.1 (depends on 3.4.1)
[error]             +- com.github.tminglei:slick-pg_2.13:0.21.1           (depends on 3.4.1)
```

**Comment by tminglei** [comment]
Hi @andyczerwonka, I have released slick-pg v0.22.2 with slick v3.5.1. Pls check it!

**Comment by andyczerwonka** [comment]
Awesome, thanks! Unfortunately we've got to migrate to Pekko (or some other strategy) before we can take advantage of it. 
![image](https://github.com/tminglei/slick-pg/assets/138898/2fb916e8-357b-4e86-bc69-84ed6e4a2aec)
